packages/suite/src/components/suite/SuiteLayout/index.tsx

```diff
@@ -11,6 +11,7 @@ import { DiscoveryProgress } from '@wallet-components';
 import NavigationBar from '../NavigationBar';
 import { useLayoutSize, useSelector, useDevice } from '@suite-hooks';
 import { MODAL } from '@suite-actions/constants';
```

Comment:
@tomasklim 'MODAL' is declared but its value is never read

Reply:
00d4fa7